<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLinesDialog</class>
 <widget class="QDialog" name="EditLinesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="nounsListWidget"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="numberOfLinesLabel">
     <property name="text">
      <string>Number Of Lines: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="editPushButton">
     <property name="text">
      <string>&amp;Edit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="removePushButton">
     <property name="text">
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="removeAllPushButton">
     <property name="text">
      <string>Remove Al&amp;l</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="donePushButton">
     <property name="text">
      <string>&amp;Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
